crates/sfpe_handbook/src/chapter_50/equation_50_2.rs

```diff
@@ -0,0 +1,29 @@
+pub fn pressure_difference_fire_compartment(t_0: f64, t_f: f64, z: f64) -> f64 {
```

Comment:
```suggestion
pub fn pressure_difference(t_0: f64, t_f: f64, z: f64) -> f64 {
```
Abiding exactly by our naming convention (which we can revisit if you like), the name should be as suggested.